modules/core/src/main/java/org/apache/ignite/internal/visor/management/ChangeManagementConfigurationTask.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.visor.management;
+
+import org.apache.ignite.IgniteException;
+import org.apache.ignite.internal.processors.gmc.ManagementConfiguration;
+import org.apache.ignite.internal.processors.gmc.ManagementConsoleProcessor;
+import org.apache.ignite.internal.processors.task.GridInternal;
+import org.apache.ignite.internal.processors.task.GridVisorManagementTask;
+import org.apache.ignite.internal.util.typedef.internal.S;
+import org.apache.ignite.internal.visor.VisorJob;
+import org.apache.ignite.internal.visor.VisorOneNodeTask;
+
+/**
+ * Sets cluster-wide configuration for management.
+ */
+@GridInternal
+@GridVisorManagementTask
+public class ChangeManagementConfigurationTask extends VisorOneNodeTask<ManagementConfiguration, ManagementConfiguration> {
+    /** */
+    private static final long serialVersionUID = 0L;
+
+    /** {@inheritDoc} */
+    @Override protected VisorJob<ManagementConfiguration, ManagementConfiguration> job(ManagementConfiguration arg) {
+        return new ChangeManagementConfigurationTask.ChangeManagementConfigurationJob(arg, debug);
+    }
+
+    /**
+     * Job that actually gets the status of rolling upgrade.
+     */
+    private static class ChangeManagementConfigurationJob extends VisorJob<ManagementConfiguration, ManagementConfiguration> {
+        /** */
+        private static final long serialVersionUID = 0L;
```

Comment:
Why we use same serialVersionUID for all Serialisable classes?

Reply:
That because we need to disable standard java checks to execute job on different versions nodes. Job class should not have any serialized field except argument/result with portable serialization.